google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadOfflineConversions.java

```diff
@@ -45,7 +45,10 @@
     @Parameter(names = ArgumentNames.GCL_ID, required = true)
     private String gclid;
 
-    @Parameter(names = ArgumentNames.CONVERSION_TIME, required = true)
+    @Parameter(names = ArgumentNames.CONVERSION_TIME, required = true,
+        description = "The date time at which the conversion occurred. "
+            + "Must be after the click time. "
+            + "The format is  \"yyyy-mm-dd hh:mm:ss+|-hh:mm\", e.g. “2019-01-01 12:32:45-08:00”.")
```

Comment:
There are some slanted quotes in here. How about using single quotes within the string instead?